src/acl/external/session/ext_session_acl.cc

```diff
@@ -222,8 +223,8 @@ static int session_active(const char *details, size_t len)
 static void
 session_login(/*const*/ char *details, size_t len)
 {
-    DB_ENTRY key = {0};
-    DB_ENTRY data = {0};
+    DB_ENTRY key = {0, 0};
+    DB_ENTRY data = {0, 0};
```

Comment:
Please double check, but I think we should use a zero-initializer that does not require knowing the details of the structure, especially if the structure is "foreign" to Squid.

```suggestion
    DB_ENTRY key = {};
    DB_ENTRY data = {};
```

If I am right, please fix all (otherwise modified) occurrences.

Reply:
Checked, it works. Adopting